compiler/modules/nback/build.bal

```diff
@@ -2045,9 +2051,9 @@ function buildBooleanCompareOp(bir:OrderOp op) returns llvm:IntPredicate {
     return <llvm:IntPredicate>unsignedIntPredicateOps[op];
 }
 
-function buildFunctionSignature(bir:FunctionSignature signature) returns llvm:FunctionType|BuildError {
-    llvm:Type[] paramTypes = from var ty in signature.paramTypes select (check semTypeRepr(ty)).llvm;
-    RetRepr repr = check semTypeRetRepr(signature.returnType);
+function buildFunctionSignature(bir:FunctionSignature signature, bir:File file) returns llvm:FunctionType|BuildError {
```

Comment:
I am not sure we can do that because to build a Scaffold we need a `llvm:FunctionDefn` and for that we need a `llvm:FunctionType` which we get from this function